gef-ui/server/controllers/enter-exporters-correspondence-address/index.js

```diff
@@ -0,0 +1,77 @@
+import * as api from '../../services/api';
+import { validationErrorHandler } from '../../utils/helpers';
+
+const enterExportersCorrespondenceAddress = async (req, res) => {
+  const { params, session } = req;
+  const { applicationId } = params;
+  const { address } = session;
+  const parseAddress = address ? JSON.parse(address) : null;
```

Comment:
Why are we storing the exporter address in a session? Won't this populate a second application they create in the same session with the exporter from the first application?

Reply:
Hope this was made clear in the stand up